MAUI/ImageEditor/shape.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Shape Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about the shape annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Shape Annotation in .NET MAUI ImageEditor (SfImageEditor)
+
+The image editor control allows you to add various shapes with customizable settings
+
```

Comment:
Add topic,

## Add shape annotation
You can annotate any shapes over an image using the [AddShape] method

Code:

imageEditor.AddShape(AnnotationShape.Arrow)

Reply:
Added a topic and updated a content